webapp/client/src/components/FormFieldDropDown.js

```diff
@@ -38,13 +38,12 @@ function FormFieldDropDown({
       {...{
         fieldName,
         errors,
-        cols: "6",
       }}
-      labelComponent=<FieldLabel {...{ label, fieldId, details }} />
+      labelComponent={<FieldLabel {...{ label, fieldId, details }} />}
```

Comment:
Why did you have to change this?

Reply:
My editor showed me that this is a syntax error. More as a warning, I guess (I don't understand Visual Studio Code's visuals yet^^). But I thought as it only makes things better, let's do it #boyscouting